### PR TITLE
NOISSUE: fix unstable test

### PR DESCRIPTION
**Pull request by ElSacramento**
fix unstable test

### CODE REVIEW

**Comment by bigbes** [comment]
please, apply that diff: 
```diff
diff --git a/ledger-core/virtual/integration/staterequest_test.go b/ledger-core/virtual/integration/staterequest_test.go
index df1ce653..39c3a40e 100644
--- a/ledger-core/virtual/integration/staterequest_test.go
+++ b/ledger-core/virtual/integration/staterequest_test.go
@@ -65,11 +65,10 @@ func TestVirtual_VStateRequest(t *testing.T) {
                        // create object
                        {
                                server.IncrementPulse(ctx)
-                               commontestutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
+                               pulseNumber = server.GetPulse().PulseNumber

                                Method_PrepareObject(ctx, server, payload.Ready, objectGlobal, pulseNumber)

-                               pulseNumber = server.GetPulse().PulseNumber
                                waitMigrate := server.Journal.WaitStopOf(&handlers.SMVStateReport{}, 1)
                                server.IncrementPulse(ctx)
                                commontestutils.WaitSignalsTimed(t, 10*time.Second, waitMigrate)
```

**Comment by ruz** [comment]
@bigbes your diff is no correct. AsOf in VStateReport should be P-1, not the current pulse and actually can be rejected by message validator.

**Comment by bigbes** [comment]
sure

File: ledger-core/virtual/integration/staterequest_test.go [hunk 1]
```diff
@@ -64,12 +64,16 @@ func TestVirtual_VStateRequest(t *testing.T) {
 
 			// create object
 			{
-				server.IncrementPulseAndWaitIdle(ctx)
+				server.IncrementPulse(ctx)
+				commontestutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
```
Comment (ruz) [inline ledger-core/virtual/integration/staterequest_test.go:68]: this change is not required!

Comment (ElSacramento) [reply ledger-core/virtual/integration/staterequest_test.go]: removed

**Comment by ElSacramento** [comment]
Fix due to fail in windows-checks
```
=== RUN   TestVirtual_VStateRequest/with_RequestLatestDirtyState_flag
    staterequest_test.go:72: 
        	Error Trace:	utils.go:50
        	            				staterequest_test.go:72
        	Error:      	timeout
        	Test:       	TestVirtual_VStateRequest/with_RequestLatestDirtyState_flag
```
https://github.com/insolar/assured-ledger/pull/585/checks?check_run_id=927617120

**Comment by ElSacramento** [comment]
Everything is alright now
```
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000 -race
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	46.280s
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	8.399s
```

File: ledger-core/virtual/integration/staterequest_test.go [hunk 2]
```diff
@@ -110,9 +111,8 @@ func TestVirtual_VStateRequest(t *testing.T) {
 			server.SendMessage(ctx, msg)
 
 			commontestutils.WaitSignalsTimed(t, 10*time.Second, waitVStateReport)
-			commontestutils.WaitSignalsTimed(t, 10*time.Second, server.Journal.WaitAllAsyncCallsDone())
 
-			require.Equal(t, 1, typedChecker.VStateReport.Count())
+			require.Equal(t, 1, typedChecker.VStateReport.CountBefore())
```
Comment (sexyhamster) [inline ledger-core/virtual/integration/staterequest_test.go:115]: I wonder why you have to use CountBefore? Handler could leak after? Why WaitAllAsyncCallsDone is removed?

Comment (ElSacramento) [reply ledger-core/virtual/integration/staterequest_test.go]: updated

**Comment by ElSacramento** [comment]
```
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000 -race
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	48.059s
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	12.727s
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000 -race
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	50.914s
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	11.426s
$ go test ./virtual/integration/ -test.run ^TestVirtual_VStateRequest$ -count=1000
ok  	github.com/insolar/assured-ledger/ledger-core/virtual/integration	12.329s
```

**Pull request merged**